### ISSUE TITLE
Error message when posting articles to my blog

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#367]
```
What's the problem?
Despite correct credentials Scribe Fire reports recently every time that a post 
could not be transmitted because the credentials were incorrect. I am using 
Scribe Fire along with the latest Opera browser.

What browser are you using?
Opera 11.11

What version of ScribeFire are you running?
1.7

```

-----
Original issue reported on code.google.com by `herbert....@online.de` on 21 Jun 2011 at 10:34

**Comment by GoogleCodeExporter** [comment]
```
What kind of blog do you have?
```

-----
Original comment by `cfinke` on 28 Jun 2011 at 1:43
* Changed state: **Pending**

**Comment by GoogleCodeExporter** [comment]
```
It's a wordpress blog.
```

-----
Original comment by `herbert....@online.de` on 28 Jun 2011 at 7:44

**Comment by GoogleCodeExporter** [comment]
-----
Original comment by `cfinke` on 24 Jul 2011 at 8:03
* Changed state: **New**

**Comment by GoogleCodeExporter** [comment]
```
fixed with latest 4.3.1 version
```

-----
Original comment by `scribefi...@gmail.com` on 14 Apr 2014 at 6:47
* Changed state: **Fixed**
